src/rust/ru.readmanga/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "readmanga"
+version = "1.0.3"
```

Comment:
This is the initial version you're adding to the repo, so it should be `1.0.0` (that being said, the cargo package name/version aren't used and don't matter atm).

Reply:
Changed back to 1.0.0